app/helpers/donations_helper.rb

```diff
@@ -0,0 +1,27 @@
+module DonationsHelper
+  def donation_payment_type_icon donation
+    case donation.payment_type.to_sym
+    when :transfer
+      image_tag "svg/transfer.svg"
+    when :payment
+      image_tag "svg/payment.svg"
+    else
+      image_tag "svg/cash.svg"
+    end
```

Comment:
```suggestion
    image_tag "svg/#{donation.payment_type}.svg"
```
viết ngắn lại kiểu này thì code có chạy k bạn?

Reply:
Ừa, được đấy bạn :v nãy fix xong quên không nhìn lại code, hợp lí đấy :v nice

Reply:
Thanks vì phát hiện hay ho này nhá :D